R/calculate.R

```diff
@@ -200,17 +200,17 @@ assume_null <- function(x, stat_) {
 
 # User supplied "too much" information - hypothesized a value for a point
 # estimate that isn't relevant to the statistic calculation
-message_on_excessive_null <- function(x, stat) {
+message_on_excessive_null <- function(x, stat = "mean", fn = "calculate") {
```

Comment:
Yes, I think it would be better to avoid the defaults (esp without comments explaining why) as it seems like it would be confusing to the next person touching this function or someone reading the source code